release-notes.md

```diff
@@ -1,5 +1,23 @@
 # Release notes
 
+## Aug 8, 2019
+
+**Places UI Update**
+
+#### New Features
+* Added a new list view showing POIs without the map
+* Added POI filtering options for city, state, country, and metadata
+* Auto create the first library in an org
+* Added POI sorting to the List View
+
```

Comment:
Please reword the 1st 3 bullets so that they read as follows:

- Added a new list view that shows POIs without the map.
- Added POI filtering options for the city, the state, the country, and the metadata.
- The first library in an organization are automatically created.